server/controllers/controllers.games.js

```diff
@@ -31,4 +31,17 @@ const getReviewById = (req, res, next) => {
     })
 };
 
-module.exports = { getCategories, getReviews, getReviewById };
\ No newline at end of file
+const getCommentsByReviewId = (req, res, next) => {
+    const id = req.params.review_id
+    const promises = [selectCommentsByReviewId(id), selectReviewById(id)]
+
+    Promise.all(promises)
```

Comment:
Great use of promise.all to validate if the ID is valid, don't worry about it for now but would be good to have a think about how you could only make the second DB call to check if the comments call returned an empty array just to optimise the code.